.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "singleQuote": true,
```

Comment:
With Prettier 2.0 we only need the singleQuote and bracketSpacing options

Reply:
I updated these settings (and ran the formatter).